include/alpaka/acc/AccCpuSycl.hpp

```diff
@@ -6,16 +6,7 @@
 
 #include "alpaka/acc/AccGenericSycl.hpp"
 #include "alpaka/acc/Tag.hpp"
-#include "alpaka/core/Concepts.hpp"
-#include "alpaka/core/DemangleTypeNames.hpp"
 #include "alpaka/core/Sycl.hpp"
-#include "alpaka/dev/DevCpuSycl.hpp"
-#include "alpaka/dev/Traits.hpp"
-#include "alpaka/kernel/TaskKernelCpuSycl.hpp"
-#include "alpaka/kernel/Traits.hpp"
-#include "alpaka/platform/PlatformCpuSycl.hpp"
-#include "alpaka/platform/Traits.hpp"
-#include "alpaka/vec/Vec.hpp"
 
 #include <string>
 #include <utility>
```

Comment:
I think these two headers can be removed (also in the other accelerator specialisations).
```suggestion
```